docs/modules/migrate/pages/evaluation-tool.adoc

```diff
@@ -0,0 +1,246 @@
+= Evaluating 5.x (For IMDG 3.x Users)
+
+Hazelcast offers an evaluation tool for IMDG 3.x users. Using it, you can try
+out some of the Hazelcast 5.x features on your existing data in the IMDG 3.x cluster.
+
+With this tool, you can:
+
+* use existing map data in 3.x and 5.x clusters as sources and sinks so that you can transfer data between them
+* use map or replicated map data from a 3.x cluster to perform stream data enrichment in a 5.x cluster
++
+See xref:pipelines:sources-sinks.adoc[Sources and Sinks] and
+xref:pipelines:map-join.adoc[Enrich a Stream] to learn more about these functionalities.
+
+The evaluation tool is basically a connector which can be used to perform the above tasks.
+
+NOTE: The evaluation tool is not intended and recommended for production usage.
+It supports only the legacy Hazelcast IMDG 3.x versions between 3.6.x and 3.12.x and Platform 5.x versions.
+
+== Get the Evaluation Tool
+
+You can find the tool at ??? (Link to GitHub repo).
```

Comment:
It's part of the full zip distribution and individual jars can be downloaded via maven